src/views/business/business-details.njk

```diff
@@ -26,9 +26,10 @@
       View and update your business details
     </h1>
 
-    <h2 class="govuk-heading-m">
-      Business contact details
-    </h2>
+    {% set hasPhoneNumbers = businessTelephone or businessMobile %}
```

Comment:
Would it be better suited to have this logic inside the presenter? The dynamic data can be unit tested then as well